addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -2167,6 +2168,15 @@ const Wysiwyg = Widget.extend({
             this.trigger_up('wysiwyg_blur');
         }
     },
+    _onScroll: function(ev) {
+        this._memoizedNodeContainsEditable = this._memoizedNodeContainsEditable ||
+            memoize(node => node.contains(this.$editable[0]));
+        const isEditableDescendantOfScrollContainer = this._memoizedNodeContainsEditable(ev.target);
+        if (isEditableDescendantOfScrollContainer) {
```

Comment:
Honestly, I find the function name plus the one-use variable a bit too much. I think it is more confusing than helpful in the sense that it makes that code look more complicated that it is. I think this would be equivalent to the following in terms of understandability, in the sense that the important name is kept:
```js
this._isEditableDescendantOfScrollContainer =
    this._isEditableDescendatOfScrollContainer ||
    memoize(scrollContainer => scrollContainer.contains(this.$editable[0]));
if (this._isEditableDescendantOfScrollContainer(ev.target)) {
```
What do you think ? Better or you prefer the version currently on the PR ?